netplugin/netd.go

```diff
@@ -160,6 +161,10 @@ func main() {
 		"cluster-store",
 		"etcd://127.0.0.1:2379",
 		"state store url")
+	flagSet.StringVar(&opts.nwDriver,
```

Comment:
Yes, the current plan is to have ovs and vpp driver code built in the same image and switch using command line option